packages/terra-consumer-layout/tests/nightwatch/PortalLayout.jsx

```diff
@@ -235,114 +235,80 @@ const data = {
           icon: 'icon-demographics',
         },
         {
-          badgeValue: null,
-          attrs: null,
           target: '_self',
           url: 'http://localhost:8080/',
           text: 'Access Logs',
           subItems: [],
-          isActive: false,
           isExternal: true,
           icon: 'icon-client-access-logs',
         },
         {
-          badgeValue: null,
-          attrs: null,
           target: '_blank',
           url: 'http://localhost:8080/',
           text: 'Help',
           subItems: [],
-          isActive: false,
           isExternal: true,
           icon: 'icon-client-question-alt',
         },
         {
-          badgeValue: null,
-          attrs: null,
           target: '_self',
           url: '#',
           isExternal: true,
           text: 'English (United States)',
-          subItems:
-          [{
-            badgeValue: null,
-            attrs: { 'data-locale': 'ar' },
-            target: '_self',
-            url: '#',
-            text: '&#8235;\u0627\u0644\u0639\u0631\u0628\u064a\u0651\u0629&#8236;',
-            subItems: [],
-            isActive: false,
-            isExternal: true,
-            icon: null,
-          },
-          {
-            badgeValue: null,
-            attrs: { 'data-locale': 'es' },
-            target: '_self',
-            url: '#',
-            text: '&#8234;Espa\u00f1ol&#8236;',
-            subItems: [],
-            isActive: false,
-            isExternal: true,
-            icon: null,
-          },
-          {
-            badgeValue: null,
-            attrs: { 'data-locale': 'en-gb' },
-            target: '_self',
-            url: '#',
-            text: '&#8234;English (United Kingdom)&#8236;',
-            subItems: [],
-            isActive: false,
-            isExternal: true,
-            icon: null,
-          },
-          {
-            badgeValue: null,
-            attrs: { 'data-locale': 'fr-fr' },
-            target: '_self',
-            url: '#',
-            text: '&#8234;Fran\u00e7ais (France)&#8236;',
-            subItems: [],
-            isActive: false,
-            isExternal: true,
-            icon: null,
-          }],
-          isActive: false,
-          icon: null }],
+          subItems: [
+            {
+              badgeValue: null,
+              // attrs: { 'data-locale': 'ar' },
+              isExternal: true,
+              url: '#1',
+              text: '\u0627\u0644\u0639\u0631\u0628\u064a\u0651\u0629',
+            },
+            {
+              badgeValue: null,
+              // attrs: { 'data-locale': 'es' },
+              isExternal: true,
+              url: '#2',
+              text: 'Espa\u00f1ol',
+            },
+            {
+              // attrs: { 'data-locale': 'en-gb' },
+              isExternal: true,
+              url: '#3',
+              text: 'English (United Kingdom)',
+            },
+            {
+              // attrs: { 'data-locale': 'fr-fr' },
```

Comment:
will these attrs work? I will need to set this on the link

Reply:
i just need to know what to with them, i commented them out since there is currently no functionality for them.